internal/controllers/printresults/print_results.go

```diff
@@ -135,6 +145,20 @@ func (pr *PrintResults) printResultsJSON() error {
 	return pr.createOutputJSON(b)
 }
 
+func (pr *PrintResults) printResultsSarif() error {
+	logger.LogInfoWithLevel(messages.MsgErrorGenerateJSONFile)
```

Comment:
i think`messages.MsgErrorGenerateJSONFile` is not what you wanted to use here

Reply:
> Oh, I think that I was using the wrong schema, this implementation is based on [this](https://docs.github.com/en/code-security/code-scanning/integrating-with-code-scanning/sarif-support-for-code-scanning#supported-sarif-output-file-properties) documentation right?

Correct. RuleID is a string rather than object. It's mostly just used for correlation between multiple files/lines which have the same violation type.

Reply:
This was a copy-paste of convention from SonarQube below, but I can change it:

https://github.com/ZupIT/horusec/blob/aaa02723eb1e8d558092185a39403bb6a21fc1bd/internal/controllers/printresults/print_results.go#L169

Reply:
This references to `messages.MsgErrorGenerateJSONFile` is inside a loop, on sonarqube we just log a info message.

What do you want here is:
- Create a new `messages.MsgInfoStartGenerateSarifFile` with a message like `messages.MsgInfoStartGenerateSonarQubeFile`
- Change the logging to `logger.LogInfoWithLevel(messages.MsgInfoStartGenerateSarifFile)`